backend/src/impl/benchmark_utils.py

```diff
@@ -278,6 +278,27 @@ def aggregate_view(
                         output_df = output_df.groupby(group_by).sum(numeric_only=True)
                     else:
                         output_df = output_df.sum(numeric_only=True)
+            elif op in {"add_default"}:
+                languages = [
+                    lang
+                    for lang in ALL_LANG
+                    if lang not in output_df[operation["column"]].values
+                ]
+                temp_df = pd.DataFrame(
+                    [[lang, 0] for lang in languages],
+                    columns=[operation["column"], "score"],
+                )
+                output_df = pd.concat([output_df, temp_df], axis=0, ignore_index=True)
+                if not by_creator:
```

Comment:
This is confusing, it seems that it's renaming the column from its actual meaning (`source_language`) to something else? I'm guessing that this is a hack to try to get the `source_language` values in the rows, but it'd be nice if we could fix this in a more fundamental way.

Reply:
Yep, I've implemented a fix, but we can discuss other potential ways of doing so in our meeting today :)